bin/cli.js

```diff
@@ -15,6 +15,17 @@
 	}
 
 	require("v8-compile-cache");
+
+	// try local module, fallback to global
+	try {
```

Comment:
@sokra you might have to do the same thing at `webpack/bin`once we publish the next branch to master. Path lookups are failing, and adjusting them according to their global/local paths aren't helping...